pobieracz_danych_gugik.py

```diff
@@ -1463,31 +1463,45 @@ def egib_excel_selected_btn_clicked(self):
             return
         egib_excel_zakres_danych = ''
 
+        rok = self.dockwidget.egib_excel_dateEdit_comboBox.currentText()
+        powiatName = self.dockwidget.egib_excel_powiat_cmbbx.currentText()
+        wojName = self.dockwidget.egib_excel_wojewodztwo_cmbbx.currentText()
+        teryt_powiat = self.dockwidget.egib_excel_powiat_cmbbx.currentData()
+        terytWoj = self.dockwidget.egib_excel_wojewodztwo_cmbbx.currentData()
+
         if self.dockwidget.powiat_egib_excel_rdbtn.isChecked():
             egib_excel_zakres_danych = 'powiat'
+            if not powiatName:
+                self.no_area_specified_warning()
+                return
+            else:
+                self.iface.messageBar().pushMessage("Informacja",
+                                            f'Pobieranie danych z Zestawień Zbiorczych EGiB dla {powiatName}({teryt_powiat}) z roku {rok}',
+                                            level=Qgis.Info, duration=10)
         elif self.dockwidget.wojew_egib_excel_rdbtn.isChecked():
             egib_excel_zakres_danych = 'wojew'
+            if not wojName:
+                self.no_area_specified_warning()
+                return
+            else:
+                self.iface.messageBar().pushMessage("Informacja",
+                                            f'Pobieranie danych z Zestawień Zbiorczych EGiB dla {wojName}({terytWoj}) z roku {rok}',
+                                            level=Qgis.Info, duration=10)
         elif self.dockwidget.kraj_egib_excel_rdbtn.isChecked():
             egib_excel_zakres_danych = 'kraj'
-
-        rok = self.dockwidget.egib_excel_dateEdit_comboBox.currentText()
-        powiatName = self.dockwidget.egib_excel_powiat_cmbbx.currentText()
-        if not powiatName:
-            self.no_area_specified_warning()
-            return
-        teryt_powiat = self.dockwidget.egib_excel_powiat_cmbbx.currentData()
-
-        self.iface.messageBar().pushMessage("Informacja",
-                                            f'Pobieranie danych z Zestawień Zbiorczych EGiB dla {powiatName}({teryt_powiat}) z roku {rok}',
+            self.iface.messageBar().pushMessage("Informacja",
+                                            f'Pobieranie danych z Zestawień Zbiorczych EGiB dla całej Polski z roku {rok}',
```

Comment:
Na przyszłość można trochę polepszyć ten zapis, ale to pewnie przy jakimś refactoringu zrobimy